packages/components/addon/components/hds/form/character-count/index.hbs

```diff
@@ -8,7 +8,6 @@
   class={{this.classNames}}
   id={{this.id}}
   {{did-insert this.onInsert}}
```

Comment:
Is this still needed? 🤔

Reply:
Unfortunately, yes - it wires up `aria-describedby`.

Reply:
Ah, yes, we need this function and the `controlId` as this is [the mechanism the field component uses to append descriptors](https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/form/field/index.hbs#L22). We have the same implementation for the [helper-text](https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/form/helper-text/index.js#L30-L39) and [error message](https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/form/error/index.js#L30-L39).

Reply:
👍